docs/en/reference/inheritance-mapping.rst

```diff
@@ -377,6 +364,21 @@ Example:
          *      )
          * })
          */
+        #[Entity]
+        #[AssociationOverrides([
```

Comment:
Would you like to delete the annotations that those attributes are supposed to replace?